Models/Spoiler.cs

```diff
@@ -19,11 +19,9 @@ public class Spoiler
 
         public List<ItemObject> ItemList { get; set; }
 
-        public ReadOnlyCollection<string> ITEM_NAMES { get; set; }
+        public int[] NewEntranceIndices { get; set; }
```

Comment:
should be NewDestinationIndices. entrance index refers specifically to the two byte index values used to spawn Link into the game world